guardrails/guardrail_globals.py

```diff
@@ -135,3 +141,36 @@ def dead_code_exclude(self):
         if self.dead_code_ignore:
             cmd = "--exclude %s" % self.dead_code_ignore
         return cmd
+
+    def read_file_to_list(self):
+        """ Function to read ignored pylint files to list """
+        ret_val = None
+        if self.lint_ignore:
+            with open(r"%s" % self.lint_ignore) as file:
+                content = file.readlines()
+            # Remove whitespace characters like `\n` at the end of each line
+            ret_val = [x.strip() for x in content]
+        return ret_val
+
+    @staticmethod
+    def list_to_str(str_list):
+        """ Function to convert ignored pylint files list to string """
+        str1 = ' '.join(str(e) for e in str_list)
+        return str1
+
+    def generate_files_lint(self):
+        """ Function to get ignored pylint files """
+        ignore_list = self.read_file_to_list()
+        res = None
+        if self.all_folders.split():
+            lint_input_list = []
+            for item in self.all_folders.split():
+                src_lint_file = glob.iglob(r'%s%s**%s*.py' % (item, os.sep, os.sep), recursive=True)
+                # src_lint_file = glob2.glob(r'%s%s**%s*.py' %(item, os.sep, os.sep)) #for 2.7
```

Comment:
Maybe you'd want to consider this for future change - could we automate selection based on the version of Python?
I feel like we have two options: 
1. Ask the user the python version in the ini file. 
2. Run python version, read the output and auto-determine. 

This way, we can abstract out the lines which are specific to python 2.x and then decide which one to run based on the detected python version?

Reply:
updated to auto determinate the python version and make the switch